### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by kaikreuzer**
Replaces #6111

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: extensions/transform/org.eclipse.smarthome.transform.regex/README.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ Now the resulting Number can also be used in the label to [change the color](htt
 
 ## Differences to plain Regex
 
-The regex is embedded in a string so when double qoutes `"` are used in an regex they need to be escaped `\"` to keep the string intact.
+The regex is embedded in a string so when double quotes `"` are used in an regex they need to be escaped `\"` to keep the string intact.
```
Comment (cweitkamp) [inline extensions/transform/org.eclipse.smarthome.transform.regex/README.md]: "... used in an regex" -> "... used in a regex"

File: extensions/transform/org.eclipse.smarthome.transform.regex/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ The special characters `\.[]{}()*+-?^$|` have to be escaped when they should be
 |---------------------------|------------------------|----------------------------|--------------------------|
 | `My network does not work.` | `s/work/cast/g` | `"My netcast does not cast."` | Replaces all matches of the string "work" with the string "cast". |
 | `My network does not work.` | `.*(\snot).*` | `" not"` | Returns only the first match and strips of the rest, "\s" defines a  whitespace. |
-| `temp=44.0'C` | `temp=(.*?)'C)`          | `44.0` | Matches whole string and retuns the content of teh captcha group `(.?)`. |
+| `temp=44.0'C` | `temp=(.*?)'C)`          | `44.0` | Matches whole string and retuns the content of the captcha group `(.?)`. |
```
Comment (cweitkamp) [inline extensions/transform/org.eclipse.smarthome.transform.regex/README.md]: "and retuns the" -> "and returns the"

File: extensions/transform/org.eclipse.smarthome.transform.regex/README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The regular expression in the format `s/<regex>/result/` (without `g`), replaces
 If the regular expression contains a [capture group](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#cg) defined by `()`, it returns the captured string. 
 Multiple capture groups can be used to retrieve multiple strings and can be combined as a result string defined in the `substitution`.
 
-The transformation can be set to be restricted to only match if the input string beginns with an character by prepending `^` to the beginning of a pattern or to only match if the input string ends with a specified charactger by appending `$` at the end.
+The transformation can be set to be restricted to only match if the input string beginns with an character by prepending `^` to the beginning of a pattern or to only match if the input string ends with a specified character by appending `$` at the end.
```
Comment (cweitkamp) [inline extensions/transform/org.eclipse.smarthome.transform.regex/README.md]: "string beginns with an character" -> "string begins with a character"

Comment (htreu) [reply extensions/transform/org.eclipse.smarthome.transform.regex/README.md]: good catch, do you mind fixing them?

**Comment by kaikreuzer** [comment]
@cweitkamp As @htreu already merged, please directly create a new PR with your fixes, thanks :-)

**Comment by cweitkamp** [comment]
Done (see ##6143)

**Pull request merged**